index.html

```diff
@@ -7,22 +7,49 @@
     <title>Awesome Books</title>
```

Comment:
The project is awesome and follows the wireframe. Please add all the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) linter config. Moreover, please add all the unnecessary files and folders like `.vscode` to `.gitignore`.

Reply:
@SajjadAhmad14 Hello, I removed the folder `.vscode`, and we already are using the correct linter config, so I am not sure what is the error from your side ...

Reply:
![Screenshot 2021-11-24 at 14 37 43](https://user-images.githubusercontent.com/31847346/143248672-522c1f16-3f56-4e03-becb-89040d774cfd.png)